src/iris/nodes/eye_properties_estimation/sharpness_estimation.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Tuple
+
+import cv2
+import numpy as np
+from pydantic import Field, validator
+
+import iris.io.validators as pydantic_v
+from iris.callbacks.callback_interface import Callback
+from iris.io.class_configs import Algorithm
+from iris.io.dataclasses import NormalizedIris, Sharpness
+
+
+class SharpnessEstimation(Algorithm):
+    """Calculate sharpness of the normalized iris.
+
+    The goal of this algorithm is to calculate the sharpness of the normalized iris using the variance of Laplacian.
+
+    LIMITATIONS:
+
+    This method may be biased against dark images with inadequate lighting.
+    """
+
+    class Parameters(Algorithm.Parameters):
+        """Parameters class for SharpnessEstimation objects.
+
+        lap_ksize (int): Laplacian kernel size, must be odd integer no larger than 31.
+        erosion_ksize (Tuple[int, int]): Mask erosion kernel size, must be odd integers.
+        """
+
+        lap_ksize: int = Field(..., gt=0, le=31)
```

Comment:
Why is the Laplacian kernel size capped at 31 ? 🙂

Reply:
this is the limitation from Opencv, cv2.Laplacian() uses sobel kernels and the maximum kernel size is 31. otherwise we get the following errors:

OpenCV(4.7.0) ...: error: (-211:One of the arguments' values is out of range) The kernel size must be odd and not larger than 31 in function 'getSobelKernels'